README.md

```diff
@@ -27,39 +29,41 @@ FHIR-based data exchange.
 
 ## Status
 
-These tests are a **DRAFT** intended to allow DTR implementers to perform 
-preliminary checks of their implementations against DTR IG requirements and provide 
-feedback on the tests. Future versions of these tests may validate other 
+These tests are a **DRAFT** intended to allow DTR implementers to perform
+preliminary checks of their implementations against DTR IG requirements and provide
+feedback on the tests. Future versions of these tests may validate other
 requirements and may change how these are tested.
 
 ## Test Scope and Limitations
 
 The DTR specification is complex and evolving and these tests do not yet
-cover the full scope of the specification. In particular, tests have been 
+cover the full scope of the specification. In particular, tests have been
 started but not yet released Light DTR EMR actors responsible for launching
 a DTR SMART App and serving data that the app can use to populate questionnaires.
 
 For the implemented actors, see suite-specific documentation on current limitations
 for the [payer server](lib/davinci_dtr_test_kit/docs/dtr_payer_server_suite_description_v201.md#limitations),
-[DTR SMART App](lib/davinci_dtr_test_kit/docs/dtr_smart_app_suite_description_v201.md#limitations), 
+[DTR SMART App](lib/davinci_dtr_test_kit/docs/dtr_smart_app_suite_description_v201.md#limitations),
 [DTR Full EHR](lib/davinci_dtr_test_kit/docs/dtr_full_ehr_suite_description_v201.md#limitations)
 tests
+[DTR Light EHR](lib/davinci_dtr_test_kit/docs/dtr_light_ehr_suite_description_v201.md#limitations)
```

Comment:
This section doesn't exist currently in the light ehr description.  Add a generic boilerplate description for this section like

```
## Limitations

The DTR IG is a complex specification and these tests currently validate conformance to only
a subset of IG requirements. Future versions of the test suite will test further
features.
```